### ISSUE TITLE
DatePicker: getting few HTTP 404 errors if locale is en_US

### ISSUE THREAD

**Issue by azakhark** [wicketstuff/core#434]
If locale is en_US and page has instance of DatePicker, I see HTTP 404 errors for following resources:
* /resources/org.wicketstuff.jwicket.ui.datepicker.DatePicker/jquery.ui.datepicker-en.js
* /resources/org.wicketstuff.jwicket.ui.datepicker.DatePicker/jquery.ui.datepicker-en.js

And following warnings in log file:

    WARN  - PackageResource            - resource [path = org/wicketstuff/jwicket/ui/datepicker/jquery.ui.datepicker-en.js, style = null, variation = null, locale = null]: Unable to find resource (status=404)
    WARN  - PackageResource            - resource [path = org/wicketstuff/jwicket/ui/datepicker/css/jquery.ui.base.css, style = null, variation = null, locale = null]: Unable to find resource (status=404)

**Comment by iamrakesh** [comment]
Issue with 404 for jquery.ui.datepicker-en.js seems to be due to https://bugs.jqueryui.com/ticket/6682
And 404 for jquery.ui.base.css, is possibly due to a typo at line#1612 in DatePicker.java.

new JQueryCssResourceReference(DatePicker.class, "css/jquery.ui.base.css")
should be
new JQueryCssResourceReference(DatePicker.class, "css/jquery.ui.core.css")

**Comment by martin-g** [comment]
Please create a Pull Request for the second issue!
For the first issue I guess we need to update the version of jQuery UI ?!
Thank you!

**Comment by iamrakesh** [comment]
Yes, Updating JQuery Ui version should fix the 404 for jquery.ui.datepicker-en.js.
PR to fix CSS 404 is https://github.com/wicketstuff/core/pull/449
